packages/config/tests/cli/snapshots/tests.js.md

```diff
@@ -1,4 +1,4 @@
-# Snapshot report for `packages/config/tests/cli/tests.js`
+# Snapshot report for `tests/cli/tests.js`
```

Comment:
I don't think this was intended?

Reply:
I think you an run `npx ava <path to test file> -u` from the root?

Reply:
legend, cheers